hollow/src/main/java/com/netflix/hollow/core/write/objectmapper/DeprecatedApi.java

```diff
@@ -0,0 +1,15 @@
+package com.netflix.hollow.core.write.objectmapper;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({FIELD, METHOD})
+public @interface DeprecatedApi {
+    String value() default "Empty Javadoc";
```

Comment:
I would omit the default and guard against `null` return for `value()` when generating the field accessor.